test/typescript/provider-add-command/test.ts

```diff
@@ -42,12 +42,8 @@ describe("provider add command", () => {
       );
       expect(res.stdout).toContain(`Package installed.`);
 
-      const packageJson = JSON.parse(driver.readLocalFile("package.json"));
-
-      expect(packageJson.dependencies).toEqual(
-        expect.objectContaining({
-          "@cdktf/provider-random": expect.any(String),
-        })
+      expect(driver.npmPackageVersion("@cdktf/provider-random")).toEqual(
+        expect.any(String)
       );
```

Comment:
Yes, but it's harder to share. We could extend the jest matchers, but that felt like too much effort in this case.

Reply:
Instead of a custom matcher we could also create a utility function that creates the matching block:
```suggestion
      expect(driver.packageJson()).toEqual(
        packageJsonWithDependency("@cdktf/provider-random")
      );
```

with this utility function
```ts
function packageJsonWithDependency(package: string, version?: string) {
  return expect.objectContaining({
    [package]: version || expect.any(String),
  });
}
```